kong/plugins/acme/schema.lua

```diff
@@ -42,6 +42,10 @@ local LEGACY_SCHEMA_TRANSLATIONS = {
   { auth = {
     type = "string",
     len_min = 0,
+    include_in_output = true,
+    translate_backwards = function(instance)
+      return instance.password
+    end,
```

Comment:
It seems we always have pairs of `include_in_output = true` plus a `translate_backwards` function. Wouldn't it be simpler to just use the presence of a `translate_backwards` as functionally equivalent to `include_in_output = true`?

Reply:
These come in pairs _here_, but don't necessarily mean the same things. We've separated them to be able to not include them in the dao response while still translating the fields. Not sure how relevant these options are in the future but should be the safer option.